### PR TITLE
docs: Updated README.md to fix spelling and grammatical errors.

### PR DESCRIPTION
**Pull request by Ayush-Prabhu**
## Changes proposed in this pull request

- I have updated README.md to correct some spelling errors and grammatically incorrect sentences.

## Context

What were you trying to do?
Corrected grammatical errors and spelling errors to improve professionalism.
## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,26 +4,26 @@
 
 ## What is Rafiki?
 
-Rafiki is open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
+Rafiki is an open source software that allows an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable [Interledger](https://rafiki.dev/concepts/interledger-protocol/overview/) functionality on its users' accounts.
```
Comment (sabineschaller) [inline README.md:7]: Can we please leave that without the "an"?

**Review by sabineschaller — COMMENTED** [review]
Rest looks good. Thank you!

**Comment by sabineschaller** [comment]
One more thing. Our CI requires our PR titles to be [conventional](https://www.conventionalcommits.org/en/v1.0.0/). Could you please add a "docs:" in the beginning of your title?

**Comment by Ayush-Prabhu** [comment]
I am sorry I don't know how to leave out the an. Thank you.

File: README.md (hunk 1 above)
Comment (Ayush-Prabhu) [reply README.md:7]: I am sorry I am unaware how to leave out the an. Thank you.

Comment (sabineschaller) [reply README.md:7]: Just don't add the "an" here, please. "Rafiki is open source software ..." is grammatically correct.

**Review by sabineschaller — APPROVED** [review]

**Comment by devcer** [comment]
Thank you for Contributing to Interledger during Hacktoberfest. If you have any accepted PRs, You can claim your Swag [here](https://community.interledger.org/devcer/hacktoberfest-swag-redemption-claim-your-cool-gear-2152). Last date to fill up the form is December 10th, 2023.  @Ayush-Prabhu

**Pull request merged**